package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Ensure package.json file is in the same directory as your server.js file. Without doing this, you will get an error.